dashboard/app/views/plc/tasks/show.html.haml

```diff
@@ -6,13 +6,13 @@
   = @task.name
 
 %p
-  %strong Associated module
-  = @task.plc_learning_module.try(:name)
+  %strong Associated modules:
+%ul
+  - @task.plc_learning_modules.pluck(:id, :name).each do |id, name|
```

Comment:
Alphabetically sort this. (We should start alphabetically sorting elements that are peers of each other)